tests/fixtures/js-stubs/config.js

```diff
@@ -43,6 +43,8 @@ export function Config(params = {}) {
     apmSampling: 1,
     dsn_requests: '',
     demoMode: false,
+    sentryUrl: 'https://sentry.io',
+    organizationUrl: 'https://foobar.sentry.io',
```

Comment:
```suggestion
    organizationUrl: 'https://foobar.us.sentry.io',
```

🤷 If we want to use values that are closer to what we'll end up with.